src/app/models/blocks.js

```diff
@@ -21,6 +21,10 @@ export const textBlock = text => ({
   ],
 });
 
+export const containerText = text => ({
```

Comment:
Possibly out of scope for this PR but it feels like as this is only used in tests and stories maybe it should exist in a test helper and not in the this directory.